packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -416,6 +425,83 @@ contract Deploy is Deployer {
         console.log("SuperchainConfig deployed at %s", address(superchainConfig));
     }
 
+    /// @notice Deploy the DelayedVetoable contract for the SuperchainConfig
+    ///         Note: Because the getters on DelayedVetoable must be read by pranking as the zero
+    ///         address, this function does not use the broadcast modifier, and instead starts and
+    ///         stops broadcasting, then starts and stops pranking, within the function body.
+    function deployDelayedVetoableForSuperchain() public {
```

Comment:
There is a `broadcast` modifier - does that not work because of the transparent nature based on `address(0)`?

Reply:
Correct, you apparently can't prank during a broadcast.